server/src/internalClusterTest/java/org/elasticsearch/cluster/allocation/ClusterRerouteIT.java

```diff
@@ -99,12 +101,12 @@ private void rerouteWithCommands(Settings commonSettings) throws Exception {
         assertThat(state.getRoutingNodes().unassigned().size(), equalTo(2));
 
         logger.info("--> explicitly allocate shard 1, *under dry_run*");
-        state = clusterAdmin().prepareReroute()
-            .setExplain(randomBoolean())
-            .add(new AllocateEmptyPrimaryAllocationCommand("test", 0, node_1, true))
-            .setDryRun(true)
-            .get()
-            .getState();
+        state = client().execute(
+            TransportClusterRerouteAction.TYPE,
+            new ClusterRerouteRequest().explain(randomBoolean())
+                .add(new AllocateEmptyPrimaryAllocationCommand("test", 0, node_1, true))
+                .dryRun(true)
+        ).actionGet(SAFE_AWAIT_TIMEOUT).getState();
```

Comment:
I see. That works because [safeGet() uses SAFE_AWAIT_TIMEOUT](https://github.com/elastic/elasticsearch/blob/b97d5cfdb746605f722dead3aff3e7a122590ded/test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java#L2217). Cool 👍